### PR TITLE
refactor queries

### PR DESCRIPTION
**Pull request by sakshibohora**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -7,7 +7,13 @@ import adminMiddleware from './src/middleware/adminMiddleware';
 import apiMiddleware from './src/middleware/apiAuth';
 import adminRoutes from './src/routes/adminRoutes';
 
-dotenv.config();
+if (process.env.NODE_ENVIRONMENT === 'prod') {
```
Comment (ashvin-kumbhani) [inline app.js:10]: Optional but it would be better if you use just `NODE_ENV`

Comment (ashvin-kumbhani) [reply app.js]: Okay

Comment (sakshibohora) [reply app.js:10]: Okay

File: dbCreate.js [hunk 2]
```diff
@@ -22,18 +22,16 @@ if (process.env.NODE_ENVIRONMENT === 'prod') {
...
+const sequelize = new Sequelize(process.env.DB_DIALECT, process.env.DB_USER, process.env.DB_PASS, {
   dialect: process.env.DB_DIALECT,
   host: process.env.DB_HOST,
 });
 
 const database = process.env.DB_NAME;
-console.log('TCL: database', database);
-// logger.info(`Creating database "${database}"...`);
 try {
   sequelize.query(`CREATE DATABASE "${database}"`).then(() => console.log('Database created'));
```
Comment (ashvin-kumbhani) [inline dbCreate.js]: let dbStatus;
try {
 dbStatus = await  sequelize.query(`CREATE DATABASE "${database}"`);
} catch(err){}

**Pull request merged**